src/event_loop.rs

```diff
@@ -262,88 +105,84 @@ impl EventLoop {
     ///
     /// [`set_control_flow()`]: ActiveEventLoop::set_control_flow()
     /// [`run_app()`]: Self::run_app()
-    #[inline]
-    #[cfg(not(all(web_platform, target_feature = "exception-handling")))]
```

Comment:
It's not that clean, but until we land https://github.com/rust-windowing/winit/pull/4165, I think you have to keep this `cfg`, otherwise it won't build?

Reply:
Ah, It'll fail to compile on a different level, thus I can not do much about it...

Maybe the `run` APIs on their own should be separated into their own Traits, so the impl will be present only when it's actually possible?